_build/index.html

```diff
@@ -347,7 +347,7 @@ <h2>Do I need to be vaccinated to attend the conference in person?</h2>
               424 38 Agnesberg<br />
               Sweden
             </p>
-            <p><a href="https://www.europython-society.org">EPS website</a></p>
+            <p><a href="https://www.europython-society.org" target="__blank">EPS website</a></p>
```

Comment:
Should it be double underscore or a single underscore? :thinking:

Reply:
dang. It should be single. IDK how I can make this silly mistake.